code/modules/vehicles/cargo_train.dm

```diff
@@ -16,10 +16,10 @@
 	var/obj/item/key/cargo_train/key
 
 /obj/item/key/cargo_train
-	name = "key"
-	desc = "A keyring with a small steel key, and a yellow fob reading \"Choo Choo!\"."
```

Comment:
Not sure why this needs changed, its a nice fun little addition that has not been complained about before nor has proven a issue.

Reply:
You know I don't like being the bad guy, in these conversations. Bit is it possible for someone to ever construct a logical argument why we should keep IC memes in the doesn't involve ooc based arguments such as "it's harmless, "it's funneh", or "this is part of our culture"?

My counterpoint just simple. Why would I make a corporation takes itself relatively seriously issue standardized equipment that contains childish jokes and referential memes from thousands of years ago? Are we trying to construct an immersive setting, or are we trying to play bald spacemen and tool boxes?

You can't have both. You can't just hold up these leftover jokes as shrines to how things used to be before we started making a real setting and trying to take ourselves seriously. They don't make sense. There's no reason for them to still be there, but we have people stubbornly wanting to keep them in because... ?